Python linters not working

<!-- bug reporting without issue template will be closed automatically -->

## Expected behavior, english is required
Python linter error messages should be displayed.

## The reproduce ways from Vim starting (Required!)
Create a simple file called linter_test.py:
```python
class TestLinter:
    pass
```
After writing the file, observe that no linter messages are displayed. However, in bash:
```
$ pylint linter_test.py
************* Module linter_test
linter_test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
linter_test.py:1:0: C0115: Missing class docstring (missing-class-docstring)
linter_test.py:1:0: R0903: Too few public methods (0/2) (too-few-public-methods)

------------------------------------
Your code has been rated at -5.00/10
```
## Debug info

Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.

It was not put into clipboard, but I saved into a file and ctrl+c,v:

<!-- please remove the issue template when request for a feature -->
## Expected behavior, english is recommend

## Environment Information

- OS: linux
- vim version: 8.1.2269
- neovim version: -
- SpaceVim version: 1.7.0-dev
- SpaceVim status: 6ce4e0e6\00

```
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

## Output of the `:SPDebugInfo!`

<details><summary> SpaceVim debug information </summary>

### SpaceVim options :

```toml
  auto_disable_touchpad = 1
  autocomplete_method = 'asyncomplete'
  autocomplete_parens = 1
  background = 'dark'
  buffer_index_type = 4
  checkinstall = 1
  colorscheme = 'gruvbox'
  colorscheme_bg = 'dark'
  colorscheme_default = 'desert'
  commandline_prompt = '➭'
  custom_color_palette = []
  custom_plugins = [['ryanoasis/vim-devicons', {'name': 'ryanoasis/vim-devicons', 'merged': 0}]]
  data_dir = '/home/csantos/.cache/'
  debug_level = 1
  default_custom_leader = '<Space>'
  default_indent = 2
  disabled_plugins = []
  enable_ale = 0
  enable_bepo_layout = 0
  enable_cursorcolumn = 0
  enable_cursorline = 1
  enable_debug = 0
  enable_googlesuggest = 0
  enable_guicolors = 0
  enable_javacomplete2_py = 0
  enable_key_frequency = 0
  enable_language_specific_leader = 1
  enable_neocomplcache = 0
  enable_neomake = 1
  enable_os_fileformat_icon = 0
  enable_powerline_fonts = 1
  enable_projects_cache = 1
  enable_statusline_bfpath = 0
  enable_statusline_mode = 0
  enable_statusline_tag = 1
  enable_tabline_filetype_icon = 1
  enable_tabline_ft_icon = 0
  enable_vimfiler_filetypeicon = 0
  enable_vimfiler_gitstatus = 1
  enable_vimfiler_welcome = 1
  enable_ycm = 0
  error_symbol = '✖'
  escape_key_binding = 'jk'
  expand_tab = 1
  filemanager = 'nerdtree'
  filetree_direction = 'right'
  filetype_icons = {}
  force_global_config = 0
  gitcommit_issue_icon = ''
  gitcommit_pr_icon = ''
  github_username = ''
  guicolors = 1
  guifont = ''
  hiddenfileinfo = 1
  home_files_number = 6
  hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'
  info_symbol = 'ⓘ'
  keep_server_alive = 1
  language = ''
  leader_guide_default_group_name = ''
  leader_guide_flatten = 1
  leader_guide_hspace = 5
  leader_guide_max_size = 0
  leader_guide_position = 'botright'
  leader_guide_run_map_on_popup = 1
  leader_guide_sort_horizontal = 0
  leader_guide_submode_mappings = {'<C-C>': 'win_close'}
  leader_guide_vertical = 0
  lint_on_save = 1
  lint_on_the_fly = 0
  max_column = 120
  plugin_bundle_dir = '/home/csantos/.cache/vimfiles/'
  plugin_manager = 'dein'
  plugin_manager_processes = 16
  plugin_name = 'coveragepy.vim'
  project_rooter_automatically = 1
  project_rooter_outermost = 1
  project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', '.SpaceVim.d/']
  projects_cache_num = 20
  realtime_leader_guide = 1
  relativenumber = 1
  search_tools = ['rg', 'ag', 'pt', 'ack', 'grep', 'findstr', 'git']
  sidebar_direction = ''
  sidebar_width = 30
  smartcloseignoreft = ['tagbar', 'vimfiler', 'defx', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal', 'SpaceVimTabsManager']
  smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']
  snippet_engine = 'neosnippet'
  src_root = 'E:\sources\'
  statusline_display_mode = 0
  statusline_inactive_separator = 'arrow'
  statusline_iseparator = 'nil'
  statusline_left_sections = ['winnr', 'filename', 'major mode', 'search count', 'syntax checking', 'minor mode lighters', 'vcs', 'hunks']
  statusline_right_sections = ['fileformat', 'cursorpos', 'percentage']
  statusline_separator = 'arrow'
  statusline_unicode_symbols = 1
  terminal_cursor_shape = 2
  todo_labels = ['@fixme', '@question', '@todo', '@idea']
  update_retry_cnt = 3
  version = '1.7.0-dev'
  vim_help_language = 'en'
  vimcompatible = 0
  warning_symbol = '⚠'
  wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'
  windows_index_type = 3
  windows_leader = 's'
  windows_smartclose = 'q'
```


### SpaceVim layers :

```toml
[[layers]]
  name="autocomplete"
  auto-completion-return-key-behavior='complete'
  auto-completion-tab-key-behavior='smart'
[[layers]]
  name="checkers"
[[layers]]
  name="format"
[[layers]]
  name="edit"
[[layers]]
  name="ui"
[[layers]]
  name="core"
[[layers]]
  name="core#banner"
[[layers]]
  name="core#statusline"
[[layers]]
  name="core#tabline"
[[layers]]
  name="shell"
  default_position='top'
  default_height=30
[[layers]]
  name="tools"
[[layers]]
  name="fzf"
[[layers]]
  name="VersionControl"
  merged=0
[[layers]]
  name="git"
  merged=0
[[layers]]
  name="default"
[[layers]]
  name="lang#python"
[[layers]]
  name="tags"
```


### SpaceVim Health checking :

SpaceVim clipboard support check report:
Checking +clipboard:
      Failed : to support +clipboard, you need recompile your vim with +clipboard support.
SpaceVim environment check report:
Current progpath: vim(/usr/bin/vim.basic)
version: 801
OS: linux
[shell, shellcmdflag, shellslash]: ['/bin/bash', '-c', 0]
SpaceVim lua support check report:
Checking +lua:
      Failed : to support +lua, you need recompile your vim with +lua support.
SpaceVim python support check report:
Checking +python3:
      SUCCEED!
Checking +python:
      Failed : to support +python, Please install vim-gtk, or build from sources.

### SpaceVim runtime log :

```log
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!
[ SpaceVim ] [09:25:52] [ Info ] Startup with argv: ['linter_test.py']
[ SpaceVim ] [09:25:52] [ Info ] Can not find project local config, start loading global config
[ SpaceVim ] [09:25:52] [ Info ] global_dir is: /home/csantos/.SpaceVim.d/
[ SpaceVim ] [09:25:52] [ Info ] start to apply config [glob]
[ SpaceVim ] [09:25:53] [ Info ] the func has been added to mode:syntax-checking
[ SpaceVim ] [09:25:53] [ Info ] the func has been added to mode:spell-checking
[   rooter ] [09:25:53] [ Info ] Load projects cache from: /home/csantos/.cache/SpaceVim/projects.json
[   rooter ] [09:25:56] [ Info ] Start to find root for: /tmp/vDFIZTi/6/issue_report.md
[   rooter ] [09:25:56] [ Info ] buffer name: /tmp/vDFIZTi/6/issue_report.md
[   rooter ] [09:25:56] [ Info ] same as current directory, no need to change.
```
</details>

## Screenshots

If you have any screenshots for this issue, please upload here. BTW you can use https://asciinema.org/ for recording video in terminal.

<!-- please remove the issue template when request for a feature -->

--- a/autoload/SpaceVim/layers/lang/python.vim
+++ b/autoload/SpaceVim/layers/lang/python.vim
@@ -141,7 +141,7 @@ function! SpaceVim#layers#lang#python#config() abort
   elseif executable('python3')
     call SpaceVim#plugins#repl#reg('python', ['python3', '-i'])
   endif
-  let g:neomake_python_enabled_makers = ['python']
+  let g:neomake_python_enabled_makers = s:enabled_linters
   let g:neomake_python_python_exe = s:python_interpreter
 endfunction
 

